scripts/lib/mkcloud-common.sh

```diff
@@ -702,4 +727,13 @@ fi
 iscloudver 8plus && : ${want_database_sql_engine:="mysql"}
 : ${want_external_ceph:=0}
 : ${want_ses_version:=0}
+
+# IPv6 Support
+: ${want_ipv6:=0}
+if [[ ${want_ipv6} == 0 ]]; then
+    : ${ip_sep:="."}
+else
+    : ${ip_sep:=":"}
```

Comment:
Did you know that you can use dots in IPv6 addrs like this?
```bash
> ping6 fd00:0:0:7::192.168.12.123
PING fd00:0:0:7::192.168.12.123(fd00::7:0:0:c0a8:c7b) 56 data bytes
```

But OTOH we might want to completely get away from IPv4

Reply:
That is completely unreadable :wink: